martin/src/cog/errors.rs

```diff
@@ -52,4 +52,9 @@ pub enum CogError {
         "The pixel size of the image {0} is not squared, the x_scale is {1}, the y_scale is {2}"
     )]
     NonSquaredImage(PathBuf, f64, f64),
+
+    #[error(
+        "Get origin failed for {0}: the length of ModelTiepointTag should be >= 6, or the length of ModelTransformationTag should be >= 12"
```

Comment:
Would this be correct?

```suggestion
        "Calculating the tile origin failed for {0}: the length of ModelTiepointTag should be >= 6, or the length of ModelTransformationTag should be >= 12"
```